magefiles/magefile.go

```diff
@@ -1302,12 +1302,17 @@ func CleanWorkload() error {
 
 func runTests(labelsToRun string, junitReportFile string) error {
 	pathToSuite := "./cmd"
+	ginkgoSeries := ""
```

Comment:
just a nit, but would `testNodes` or `testProcesses` var name be more descriptive?